driver/driver.go

```diff
@@ -54,17 +44,25 @@ func New(url string) (Driver, error) {
 		return nil, err
 	}
 
-	if d, found := driverMap[u.Scheme]; found {
+	driver := registry.GetDriver(u.Scheme)
+	if driver != nil {
+		blankDriver := reflect.New(reflect.TypeOf(driver)).Interface()
```

Comment:
Agree. I just took @lepoetemaudit 's pull request, and fixed the tests. I'll try to come up with something next week.
Feel free to contribute.

Thanks
Philippe